ci-operator/step-registry/platform-external/cluster/aws/install/platform-external-cluster-aws-install-commands.sh

```diff
@@ -0,0 +1,450 @@
+#!/bin/bash
```

Comment:
TODO check if we can move this to a generic upi flow (currently it has aws type bits)